From: Lubomir Rintel <lkundrak@v3.sk>
Date: Sat, 11 Aug 2018 11:08:42 +0200
Subject: utils/test: don't assert on debug level messages

They come and go in GLib core for all sorts of purposes. Don't let that
break our tests.

(cherry picked from commit 3e8eef538987c909e87b871a6b8c74aee08661a3)
Origin: upstream, 1.12.4, commit:5810c46b118447567e52199ca139c7a5e9be6ab4
---
 shared/nm-utils/nm-test-utils.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

Index: network-manager-1.12.2/shared/nm-utils/nm-test-utils.h
===================================================================
--- network-manager-1.12.2.orig/shared/nm-utils/nm-test-utils.h
+++ network-manager-1.12.2/shared/nm-utils/nm-test-utils.h
@@ -573,8 +573,9 @@ __nmtst_init (int *argc, char ***argv, g
 #endif
 	} else {
 		/* We were called not to set logging levels. This means, that the user
-		 * expects to assert against (all) messages. Any uncought message is fatal. */
-		g_log_set_always_fatal (G_LOG_LEVEL_MASK & ~G_LOG_LEVEL_DEBUG);
+		  * expects to assert against (all) messages.
+                  * Any uncaught message on >debug level is fatal. */
+                g_log_set_always_fatal (G_LOG_LEVEL_MASK & ~G_LOG_LEVEL_DEBUG);
 	}
 
 	if ((!__nmtst_internal.assert_logging || (__nmtst_internal.assert_logging && __nmtst_internal.no_expect_message)) &&
